draft-dhcwg-dhc-rfc8415bis.xml

```diff
@@ -1,6738 +0,0 @@
-<?xml version="1.0" encoding="US-ASCII"?>
```

Comment:
I agree that keeping copies in git is not a good practice. But I'm against removing them the way it's done here.

In my perfect world, there would be one file with history and specific versions would be tagged.

You might also check up with @bevolz. I recall Bernie was fond of keeping the copies for easier comparison.

Reply:
Sure, we can use GIT as a shared zip drive.  But, the extra files distract and confuse.

Reply:
I think you missed my point. I'm OK with removing them, just not the way how you proposed by simply deleting them. This way you'd lose the history of changes, you wouldn't be able to use `git blame` to find out who committed specific line, etc.

I also agree that keeping multiple copies is confusing. I did apply my changes to wrong file...

I don't have a good answer how to handle this properly, just some vague idea that could possibly work. We could do interactive rebase to rewrite history. We could keep renaming a single file to match whatever filename the next commit wanted to update. At the end, we would have a single file with the changes applied by appropriate authors. It's a bit more work. I can do an experiment some time next week if you feel it's useful.

Reply:
> I think you missed my point. I'm OK with removing them, just not the way how you proposed by simply deleting them. 

I have now merged them into the history.